app/helpers/repo_helper.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module RepoHelper
+  require 'rest-client'
+  require 'json'
+
+  def get_projects_of_user(user, page)
+    all_user_projects_url = "https://gitlab.com/api/v4/users/#{user.gid}"\
+      "/projects?page=#{page}&per_page=9"
+    headers = {
+      'Accept' => 'application/json',
+      'Authorization' => "Bearer #{user.access_token}"
+    }
+    rsp = RestClient.get(all_user_projects_url, headers)
+    pagination = {
```

Comment:
Ideally, keep pagination and fetching projects into two different parts. Fetching projects is part of business logic. Pagination is view logic. That way we can adhere to SRP